posix/posix.c

```diff
@@ -647,9 +647,10 @@ int32_t red_format(
 
     <b>Errno values</b>
     - #RED_EINVAL: Volume is not mounted; or @p pszVolume is `NULL`.
-    - #RED_EIO: I/O error during the transaction point.
+    - #RED_EIO:    I/O error during the transaction point.
     - #RED_ENOENT: @p pszVolume is not a valid volume path prefix.
     - #RED_EUSERS: Cannot become a file system user: too many users.
+    - #RED_EROFS:  The file system volume is read-only.
```

Comment:
Adding alignment here is inconsistent, since the errno documentation for other POSIX-like APIs isn't aligned.

Reply:
OK